tools/update_vendor.sh

```diff
@@ -12,6 +12,10 @@ fi
 # update vendor/
 go mod vendor
 # remove repo-originated BUILD files
-find vendor/ -name BUILD | xargs rm
+find vendor -type f \( \
```

Comment:
Consider copying https://github.com/kubernetes/test-infra/blob/master/hack/update-deps.sh and https://github.com/kubernetes/test-infra/blob/master/hack/verify-deps.sh